### ISSUE TITLE
 Is the data division of 8: 1: 1 used to stop the training of the model with one of them as the validation set?

### ISSUE THREAD

**Issue by czjczj** [masashitsubaki/CPI_human#8]
Is the data division of 8: 1: 1 used to stop the training of the model with one of them as the validation set?

**Comment by czjczj** [comment]
I don't see any effect of one of the validation sets from the code. I think it is used to stop the model training and prevent the model from overfitting.
